### ISSUE TITLE
Can't enable SSH access: "Running SSH server is not permitted"

### ISSUE THREAD

**Issue by florian-obradovic** [netbirdio/netbird#2632]
**Describe the problem**

I tried to enable SSH Access for a few of my peers (Linux & macOS) but it doesn't work

**To Reproduce**

1. Enable SSH Access for a peer in dashboard
2. client.log on the peer shows: **WARN client/internal/engine.go:554: running SSH server is not permitted**
3. try to connect connect:
```
Error: dial tcp 100.102.98.127:44338: i/o timeout
Couldn't connect. Please check the connection status or if the ssh server is enabled on the other peer
```

**Expected behavior**
- Enable SSH access

**Are you using NetBird Cloud?**
Self hosted!

**NetBird version**
`0.29.3`

**NetBird status -dA output:**

OS: linux/amd64
Daemon version: 0.29.3
CLI version: 0.29.3
Management: Connected to https://netbird.anon-ZsVFN.domain:33073
Signal: Connected to http://netbird.anon-ZsVFN.domain:10000
Relays:
  [stun:netbird.anon-ZsVFN.domain:3478] is Available
  [turn:netbird.anon-ZsVFN.domain:3478?transport=udp] is Available
Nameservers:
FQDN: anon-poKed.domain
NetBird IP: 100.102.98.127/16
Interface type: Kernel
Quantum resistance: false
Routes: -
Peers count: 5/11 Connected

**Screenshots**
![CleanShot 2024-09-23 at 13 54 03@2x](https://github.com/user-attachments/assets/5ea6c9ab-60d7-4b5a-baab-4710a63c5ab8)

**Comment by mlsmaycon** [comment]
Hi @florian-obradovic, you need to run this on the client:
```shell
netbird down
netbird up --allow-server-ssh
```
We will update our docs since this information is missing.

**Comment by sirvar** [comment]
@mlsmaycon has this been updated in the docker image as well?

**Comment by mlsmaycon** [comment]
> ```shell
> allow-server-ssh
> ```

Yes its been @sirvar. For docker you need the following environment variable:

```
NB_ALLOW_SERVER_SSH=true
```

**Comment by florian-obradovic** [comment]
Thanks for heads up @mlsmaycon 

We should add a note to the documentation, that you also need an ACL which allows tcp port 44338 access.
`2024-09-25T23:20:12+02:00 INFO client/ssh/server.go:248: starting SSH server on addr: 100.102.98.127:44338`

**Comment by creeram** [comment]
@mlsmaycon  Why is the allow ssh from the dashboard not working? it works only when enabled from cli.

**Comment by lixmal** [comment]
@creeram It's a security feature

@florian-obradovic the rule is not required in 0.35+

**Comment by creeram** [comment]
@lixmal  security feature? If it's not possible to allow ssh from the dashboard why do we have that option?

**Comment by jakob1379** [comment]
for me the issue was that my netbird config is not at the default location, but for some reason the location is not pulled from the daemon.

This means I have to do 
```bash
sudo netbird -c path/to/config ssh my-remote
# or alternatively
export NETBIRD_CONFIG=path/to/config
sudo -E netbird ssh my-remote
```

**Comment by mlsmaycon** [comment]
closing issue due to no recent feedback. Feel free to open a new one if the issue persist or reopen if this was a feature request.
